src/tools/auth0/handlers/emailProvider.ts

```diff
@@ -16,7 +17,8 @@ export default class EmailProviderHandler extends DefaultHandler {
 
   async getType(): Promise<Asset> {
     try {
-      return await this.client.emailProvider.get({ include_fields: true, fields: defaultFields });
+      const { data  } = await this.client.emails.get({ include_fields: true, fields: defaultFields.join(',') });
```

Comment:
@adamjmcgrath This is a breaking change we want to call out.

Reply:
https://github.com/auth0/node-auth0/blob/beta/v4_MIGRATION_GUIDE.md#response-change

Reply:
Ah, sorry - this is from our chat about `fields` - ignore me....